helm/reana/README.md

```diff
@@ -34,6 +34,7 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `components.reana_workflow_controller.image`             | [REANA-Workflow-Controller image](https://hub.docker.com/r/reanahub/reana-workflow-controller) to use | `reanahub/reana-workflow-controller:<chart-release-version>` |
 | `components.reana_workflow_controller.imagePullPolicy`   | REANA-Workflow-Controller image pull policy                                          | IfNotPresent                                    |
 | `components.reana_workflow_controller.environment.REANA_JOB_HOSTPATH_MOUNTS`   | JSON list of optional hostPath mounts, for all user jobs. Each mount object has a key `name` (name of the mount), `hostPath` (path to the directory to be mounted from the Kubernetes nodes) and `mountPath` (path inside the job containers where the `hostPath` will be mounted)            | None                                            |
+| `components.reana_workflow_controller.environment.REANA_KUBERNETES_JOBS_KEEP_ON_STATUS_LIST` | Keep Kubernetes jobs (workflow engines and jobs) with the configured statuses alive e.g. `"finished,failed"`. Possible statuses `reana_db.models.WorfkflowStatus` and `reana_db.models.JobStatus` see in [REANA-DB documentation](https://reana-db.readthedocs.io/en/latest/). | None |
```

Comment:
The link to REANA-DB cannot be to the correct version, shall I just switch to point to a permalink to the file on GitHub? or just mention REANA-DB docs without a link? The best would definitely be to have a link to `0.7` docs.

Reply:
`https://reana-db.readthedocs.io/en/stable/` apparently points to `0.7.0` but I'm not sure if it's something automatic or configured by us. Shouldn't it point to `0.7.1` instead?

Reply:
Forced the rebuilt of `stable`, now it is 0.7.1 indeed.

(That said, we can activate builds on tags, so that they would appear explicitly named as such...)

Reply:
... example: https://reana-db.readthedocs.io/en/0.7.1/

Reply:
Having docs pointing to tags looks great. OTOH, if we point to a specific version in this file we take the risk of it getting outdated when we release e.g `0.8.0`.